lisa/sut_orchestrator/libvirt/schema.py

```diff
@@ -114,3 +114,6 @@ class CloudHypervisorNodeSchema(BaseLibvirtNodeSchema):
     # Can be obatained from:
     # https://github.com/cloud-hypervisor/rust-hypervisor-firmware
     firmware: str = ""
+    # guest_vm_type can be NON-CVM or CVM
+    guest_vm_type: str = "NON-CVM"
```

Comment:
It should be set by the feature SecurityProfile.

Reply:
@squirrelsc This is just pipeline parameter used to differentiate what type of libvirt xml we want to create. It has nothing to do with azure vm security profile. Azure VM here will be standard VM only.

Also this is for guest vm (L2 guest vm), not for L1 vm (azure vm).

Reply:
The features folder contains abstract features, they are not for Azure only. The features are for test cases running across platforms. For example, if a CVM test case can run on both Azure and libvirt, the feature can filter and setup right environments for the test case.

Reply:
Hi Chi, i have removed that commit and will create separate PR for it.

Reply:
No problem. Please wait for reviewing from @cwize1